src/events/_event-post-macros.html

```diff
@@ -268,31 +268,48 @@ <h2 class="event-venue_heading">{{ city }}</h2>
                     ) }}
                 </div>
                 <div class="content-l_col content-l_col-1-2 event-calendar_container">
-                    {% if (event_state == 'past') %}
+                    {% if (event_state == 'past' and post.archive.youtube is not defined) %}
                         Video Replay will be<br> available shortly
                     {% else %}
                         {{ event_meta_datetime(post.beginning_time) }}
-                        <!-- TDOD: Replace with real download link -->
+                        {% if event_state == 'future' %}
+                        <!-- TODO: Replace with real download link -->
```

Comment:
Trivial: prefer jinja comments (`{# TODO: #}`) so TODOs don't end up in page's output source.